db/schema.rb

```diff
@@ -11,20 +11,20 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160701215002) do
```

Comment:
version updates are fine

On Tue, Jul 12, 2016 at 7:19 AM, Ben Straub <notifications@github.com>
wrote:

> In db/schema.rb
> <https://github.com/zendesk/samson/pull/1130#discussion_r70447316>:
>
> > @@ -11,20 +11,20 @@
> >  #
> >  # It's strongly recommended that you check this file into your version control system.
> >
> > -ActiveRecord::Schema.define(version: 20160701215002) do
>
> I rebased out most of these changes, leaving just the two table
> definitions. Is it good practice to avoid updating the schema version as
> well?
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1130/files/2240ef77f3386f8e2f818a32d96356c4a741e711#r70447316>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAsZ0lSWCBai4DAKQK6VyIE2qmJuwyEks5qU6JXgaJpZM4JIRHk>
> .
>